packages/web/src/components/pages/TestSuites/TestSuitesList/TestSuitesList.tsx

```diff
@@ -39,7 +41,7 @@ const TestSuitesList: FC = () => {
     isFetching,
   } = useGetTestSuitesQuery(filters || null, {
     pollingInterval: PollingIntervals.everySecond,
-    skip: !isAvailable,
+    skip: !isSystemAvailable || (!isAvailable && Boolean(storeTestSuites)),
```

Comment:
Should it avoid querying when there agent is down and some test suites are loaded? I thought that we ended up keeping the polling behavior, in case of i.e. ephemeral environments